data/build.gradle.kts

```diff
@@ -106,8 +106,9 @@ class ApiKeyProperties(pathToProperties: String, project: Project) {
 }
 
 dependencies {
+    implementation(project(mapOf("path" to ":domain")))
 // TODO: Find a way to make sure we are aware of out-of-date versions of any static aars/jars in /libs. Manually check for any updates at/prior to dev signoff.
-    implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
+//     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
```

Comment:
@BottleRocket-Colin Same comment as I left on the `app/build.gradle.kts` regarding addition of another TODO comment